.github/workflows/pkgci_test_sharktank.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           repository: iree-org/iree-test-suites
-          ref: c644a9dfc3e5e1a9d071d5e786b79cf612e9b1d3
+          ref: dab5402da679e713b94a8bd5c400a51cce0e665a
```

Comment:
Diff: https://github.com/iree-org/iree-test-suites/compare/c644a9dfc3e5e1a9d071d5e786b79cf612e9b1d3...dab5402da679e713b94a8bd5c400a51cce0e665a

(Would be nice if GitHub would do that for us... oh well)